jest/genConfig.js

```diff
@@ -0,0 +1,79 @@
+/*
+  Generates configuration for jest.
+ */
+var fs = require('fs');
+var path = require('path');
+var pluginsFolder = process.env.DCOS_UI_PLUGINS;
+var testPaths = ['src'];
+if (pluginsFolder) {
+  try {
+    require('../' + pluginsFolder);
```

Comment:
We need to check the path is valid. If it isn't a valid path then we don't add the path to jest because jest won't be able to resolve it. The next line will never execute.